### PR TITLE
pipeline(ticdc): disable table actor

### LINKED ISSUE #5726: disable table actor by default
table actor pipeline implementation does not improve the ticdc performance as expected,  so disable it in release-6.1

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #5726

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add 

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by asddongmen — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]
LGTM

File: cdc/sink/mysql/mysql.go [hunk 1]
```diff
@@ -431,6 +443,9 @@ func (s *mysqlSink) createSinkWorkers(ctx context.Context) error {
 }
 
 func (s *mysqlSink) notifyAndWaitExec(ctx context.Context) {
+	defer func() {
+		log.Info("notifyAndWaitExec finished")
```
Comment (sdojjy) [inline cdc/sink/mysql/mysql.go:447]: maybe we should use debug level log

File: cdc/sink/mysql/mysql.go [hunk 2]
```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"go.uber.org/atomic"
```
Comment (hi-rustin) [inline cdc/sink/mysql/mysql.go:20]: Please refine this order.

File: cdc/sink/mysql/mysql.go [hunk 3]
```diff
@@ -453,7 +468,9 @@ func (s *mysqlSink) notifyAndWaitExec(ctx context.Context) {
 	// scenario happens, the blocked goroutine will be leak.
 	select {
 	case <-ctx.Done():
+		return
```
Comment (hi-rustin) [inline cdc/sink/mysql/mysql.go:471]: we can remove this?

**Review by hi-rustin — APPROVED** [review]
rest LGTM

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**